tests/upgrades/test_foreman_maintain.py

```diff
@@ -16,25 +16,21 @@
 
 :Upstream: No
 """
-from unittest2.case import TestCase
 from upgrade_tests import post_upgrade
 from upgrade_tests import pre_upgrade
 
 from robottelo import ssh
 
 
-class ScenarioForemanMaintain(TestCase):
+class TestForemanMaintain:
```

Comment:
Is there a need to keep the class? If not, you can remove it and make all tests module-level

Reply:
Yes, it's kinda doable but here with this PR trying to keep the same structure for all tests under `/tests/upgrades`, like keeping a class, and tests inside a class.

Reply:
I later saw a discussion where I believe @jyejare said classes were required for upgrade tests.

We can leave as-is for now, but I'd like to eventually come back around and address that limitation.